core/src/main/java/com/linecorp/armeria/server/file/HttpFileServiceBuilder.java

```diff
@@ -98,11 +102,38 @@ public HttpFileServiceBuilder clock(Clock clock) {
     }
 
     /**
-     * Sets the maximum allowed number of cached file entries. If not set, {@value #DEFAULT_MAX_CACHE_ENTRIES}
+     * Sets the maximum allowed number of cached file entries. If not set, {@code DEFAULT_ENTRY_CACHE_SPEC}
```

Comment:
I think this is not user friendly. How about just `{@code 1024}` at the expense of duplication?